nbval/plugin.py

```diff
@@ -69,16 +71,31 @@ def pytest_addoption(parser):
                          'the same enviornment that py.test was '
                          'launched from.')
 
+    term_group = parser.getgroup("terminal reporting")
+    term_group._addoption(
+        '--nbdime', action='store_true',
+        help="view failed nbval cells with nbdime.")
+
+
+def pytest_configure(config):
+    config.option.verbose -= config.option.quiet
```

Comment:
I think this got included in a copy/paste from somewhere else. I'm going to guess it was an example for adding a quiet option. I'll remove it.